src/kubelet/kubelet_test.go

```diff
@@ -118,8 +110,6 @@ func kubeletExclusions() []exclude.Func {
 				return limitsRequestsRegex.MatchString(spec.Name)
 			},
 		),
-		// Exclude metrics that depend on limits when those limits are not set.
-		exclude.Exclude(exclude.Groups("pod", "container"), exclude.Dependent(utilizationDependencies)),
```

Comment:
This line was intended to exclude utilization metrics only when pod does not have request/limits, should we really remove it?

Reply:
That is not needed anymore, I can bring it back in case it is needed in the future :)

Reply:
@roobre let me know what you think

Reply:
I'm not sure I understand, why is it not needed anymore 🤔 

If I have a pod without `requests.limits.memory` (in k8s), what would the value of `memoryUtilization` be?

Reply:
that in the current scenario it is not needed. We do not need that exclude rule to pass tests with the current dataset

Reply:
That should not be the case, we should miss this metrics as many (most) of the containers have no `resources` specified.

I think there's a bug on the not running exclusion, I'm working on it now.

Reply:
Done, just pushed a commit to make the test fail 😈 @paologallinaharbur